internal/export/exportfile.go

```diff
@@ -131,30 +132,66 @@ func unmarshalContent(file *zip.File) (*export.TemporaryExposureKeyExport, error
 	return message, nil
 }
 
-func marshalContents(eb *model.ExportBatch, exposures []*publishmodel.Exposure, batchNum int32, batchSize int32, signers []*Signer) ([]byte, error) {
+func sortExposures(exposures []*publishmodel.Exposure) {
+	sort.Slice(exposures, func(i, j int) bool {
```

Comment:
Should we use `SliceStable`? Can there even be duplicates?

Reply:
no, there can't be duplicates in this slice.